src/vllm_tgis_adapter/grpc/adapters.py

```diff
@@ -134,17 +139,35 @@ def _load_adapter_config_from_file(adapter_id: str, adapter_path: str) -> dict:
             adapter_id, "directory does not exist"
         )
 
+    # 🌶️🌶️🌶️ Check for caikit-style adapters first
+    if Path(adapter_path).exists() and (Path(adapter_path) / "decoder.pt").exists():
```

Comment:
```suggestion
    if (Path(adapter_path) / "decoder.pt").exists():
```

Wouldn't this be enough?

Reply:
you right ❤️